app/src/main/java/com/kylecombes/micrepeater/Information.java

```diff
@@ -0,0 +1,39 @@
+package com.kylecombes.micrepeater;
+
+import android.media.Image;
+import android.os.Bundle;
+import android.text.method.LinkMovementMethod;
+import android.util.DisplayMetrics;
+import android.view.View;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+public class Information extends MainActivity {

```

Comment:
Should name the class `InformationActivity` and only extend `Activity` (not `MainActivity`). We don't want to extend `MainActivity` here because they are not the same.